src/main/java/com/puppycrawl/tools/checkstyle/utils/ModuleReflectionUtils.java

```diff
@@ -74,7 +75,8 @@ public static boolean isCheckstyleModule(Class<?> clazz) {
                     || isFileSetModule(clazz)
                     || isFilterModule(clazz)
                     || isFileFilterModule(clazz)
-                    || isRootModule(clazz));
+                    || isRootModule(clazz)
+                    || isTreeWalkerFilterModule(clazz));
```

Comment:
Please move this closer to the other 2 filters.
Also please update javadoc to include the new module.